src/components/unapprovedImagesTable/index.tsx

```diff
@@ -55,34 +135,116 @@ function responseToTableData(
     dateSubmitted: data.dateSubmitted,
     species: data.commonName,
     neighborhood: NEIGHBORHOOD_IDS[data.neighborhoodId],
+    submittedBy: data.uploaderName,
+    address: data.address,
   };
 }
 
 const UnapprovedImagesTable: React.FC<UnapprovedImagesTable> = ({
   fetchData,
   setSelectedImageIds,
+  useGridView,
+  approvedOrRejectedImageIds,
+  setApprovedOrRejectedImageIds,
 }) => {
+  const filteredData = useMemo(
+    () =>
+      fetchData
+        ? fetchData.filter(
+            (data) => !approvedOrRejectedImageIds.includes(data.imageId),
```

Comment:
Just to make sure I understand, whenever someone accepts or rejects an image, after closing the approval box modal the images should disappear from grid view without refreshing?

Reply:
I might just be missing it, but in table view does approving/rejecting also update this `approvedOrRejectedImageIds` list? If not, then the changes made in table view might not be reflected in grid view if not refreshed.